### PR TITLE
Add missing french translations for OM

### PR DESCRIPTION
**Pull request by Arthur-GYT**
I'm opening the PR as a draft while I do some translations for the website.

### CODE REVIEW

File: data/strings/strings.txt [hunk 1]
```diff
@@ -28817,6 +28819,7 @@
     tags = android
     en = https://organicmaps.app/faq/text-to-speech-android-tts/
     es = https://organicmaps.app/es/faq/instrucciones-de-voz-tts-en-android/
+    fr = https://organicmaps.app/fr/faq/synthèse-vocale-tts-sur-android
```
Comment (strump) [inline data/strings/strings.txt:28822]: This link is invalid. English page https://organicmaps.app/faq/text-to-speech-android-tts/ is not yet translated to French. You can leave English page link or https://organicmaps.app/fr/faq/voice/cant-hear-instructions/

Comment (biodranik) [reply data/strings/strings.txt:28822]: Maybe it's better to translate the related FAQ page to French and then have the good link? )

https://github.com/organicmaps/organicmaps.github.io/tree/master/content/faq/voice/text-to-speech-android-tts

Comment (Arthur-GYT) [reply data/strings/strings.txt:28822]: > This link is invalid. English page https://organicmaps.app/faq/text-to-speech-android-tts/ is not yet translated to French. You can leave English page link or https://organicmaps.app/fr/faq/voice/cant-hear-instructions/

https://github.com/organicmaps/organicmaps.github.io/pull/250

**Comment by Arthur-GYT** [comment]
@Jean-BaptisteC Any feedback ?

File: iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings
Comment (Jean-BaptisteC) [inline iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings:1]: Please remove this change

**Review by Jean-BaptisteC — APPROVED** [review]
LGTM

File: android/app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -1194,6 +1202,7 @@
 	<string name="type.healthcare.speech_therapist">Logopédie</string>
 
 	<!-- SECTION: Types: Roads -->
+	<string name="type.highway">Autoroute</string>
```
Comment (Jean-BaptisteC) [inline android/app/src/main/res/values-fr/strings.xml:1205]: tag without value is never used

File: android/app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -1400,6 +1409,7 @@
 	<string name="type.historic.wayside_cross">Croix de chemin</string>
 	<string name="type.historic.wayside_shrine">Oratoire</string>
 	<string name="type.historic.wreck">Naufrage</string>
+	<string name="type.junction">Jonction</string>
```
Comment (Jean-BaptisteC) [inline android/app/src/main/res/values-fr/strings.xml:1412]: Ditto

File: android/app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -1586,6 +1598,7 @@
 	<string name="type.power.plant.hydro">Centrale hydroélectrique</string>
 	<string name="type.power.plant.solar">Centrale solaire</string>
 	<string name="type.power.plant.wind">Ferme d\'éoliennes</string>
+	<string name="type.power.station">Centrale électrique</string>
```
Comment (Jean-BaptisteC) [inline android/app/src/main/res/values-fr/strings.xml:1601]: This value was deprecated

File: android/app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -2046,22 +2061,29 @@
...
 	<string name="type.leisure.resort">Complexe touristique</string>
 	<string name="type.tourism.theme_park">Parc d\'attractions</string>
 	<string name="type.tourism.viewpoint">Belvédère</string>
 	<!-- Typically more basic, not staffed and free (compared to alpine_hut). -->
 	<string name="type.tourism.wilderness_hut">Refuge non gardé</string>
+	<string name="type.tourism.zoo">Zoo</string>
 	<string name="type.tourism.zoo.petting">Zoo pour enfants</string>
 	<string name="type.traffic_calming.bump">Ralentisseur</string>
 	<string name="type.traffic_calming.hump">Ralentisseur</string>
+	<string name="type.waterway">Cours d\'eau</string>
```
Comment (Jean-BaptisteC) [inline android/app/src/main/res/values-fr/strings.xml:2076]: tag without value or with value yes is never used

File: android/app/src/main/res/values-fr/strings.xml [hunk 6]
```diff
@@ -2046,22 +2061,29 @@
...
+	<string name="type.waterway.canal">Canal</string>
+	<string name="type.waterway.canal.tunnel">Tunnel du canal</string>
 	<string name="type.waterway.fish_pass">Échelle à poissons</string>
 	<string name="type.waterway.dam">Barrage</string>
 	<string name="type.barrier.ditch">Fossé</string>
 	<string name="type.natural.water.ditch">Fossé de drainage</string>
 	<string name="type.waterway.ditch.tunnel">Drain</string>
 	<string name="type.waterway.dock">Cale</string>
+	<string name="type.waterway.drain">Drainage</string>
```
Comment (Jean-BaptisteC) [inline android/app/src/main/res/values-fr/strings.xml:2085]:
```suggestion
	<string name="type.waterway.drain">Drain</string>
```

File: android/app/src/main/res/values-fr/strings.xml [hunk 7]
```diff
@@ -2046,22 +2061,29 @@
...
+	<string name="type.natural.water.drain">Drainage</string>
```
Comment (Jean-BaptisteC) [inline android/app/src/main/res/values-fr/strings.xml:2086]:
```suggestion
	<string name="type.natural.water.drain">Drain</string>
```

**Review by Jean-BaptisteC — CHANGES_REQUESTED** [review]

**Pull request closed without merging**